packages/alfa-dom/test/get-specificity.spec.ts

```diff
@@ -51,3 +50,10 @@ test("A class is more specific than a pseudo-element-selector", t => {
       getSpecificity(selector("::first-letter"))
   );
 });
+
+test("A compound selector is more specific than a class", t => {
```

Comment:
Not always true; `div::before` is, for example, _less_ specific than `.foo` despite it being a compound selector.